client/state/sites/selectors/is-jetpack-site-multi-site.ts

```diff
@@ -0,0 +1,13 @@
+import getRawSite from 'calypso/state/selectors/get-raw-site';
+import { IAppState } from 'calypso/state/types';
+import isJetpackSite from './is-jetpack-site';
+
+export default function isJetpackSiteMultiSite( state: IAppState, siteId: number | null ) {
```

Comment:
It is better to retain the JSDoc comment.

Reply:
I don't fully understand what this comment is trying to community (probably some code changed since it was written?)

Either way, when working with mixes of JSDoc and TypeScript types I find it most useful to eliminate the typing from the JSDoc since that introduces split realities and nothing enforces the JSDoc soundness while TypeScript enforces the type annotation soundness.

```ts
/**
 * …
 *
 * @param state  Global state tree
 * @param siteId Site ID
 * @returns true if the site is multi-site
```

TypeScript plays well with JSDoc, and when we have no type annotations, the types in the JSDoc are useful, but when we have two parallel descriptions of the types it gets confusing.